go.mod

```diff
@@ -0,0 +1,22 @@
+module github.com/containerd/aufs
+
+go 1.13
+
+require (
+	github.com/Microsoft/hcsshim v0.8.7 // indirect
+	github.com/containerd/containerd v1.3.2
```

Comment:
vendor master as in https://github.com/containerd/zfs/pull/26

Reply:
Keep the deps consistent with zfs now.

Reply:
Works great, thanks!